validation-test/compiler_crashers_2_fixed/parser-cutoff.swift

```diff
@@ -0,0 +1,793 @@
+func foo(a: Int) {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+if a > 1 {
+    
```

Comment:
I assume this was from me, obviously doesn't matter but could remove if you wanted it to be a little neater :P

Reply:
Whatever. The file hit the issue before and now doesn’t hit it any longer so I don’t want to change it anymore.